proposal/indented-syntax-improvements.md

```diff
@@ -0,0 +1,215 @@
+# Indented Syntax Improvements
+
+*([Issue](https://github.com/sass/sass/issues/216))*
+
+This proposal improves the indented syntax format, allowing multiline
+expressions, semicolons, and curly braces.
+
+## Table of Contents
+
+* [Background](#background)
+* [Summary](#summary)
+  * [Ending statements](#ending-statements)
+  * [Places where a line break must create a statement break](#places-where-a-line-break-must-create-a-statement-break)
+    * [After a SassScript value](#after-a-sassscript-value)
+    * [After a non-enclosed list begins](#after-a-non-enclosed-list-begins)
+    * [At-rules](#at-rules)
+  * [Design Decisions](#design-decisions)
+* [Syntax](#syntax)
+  * [IndentedStatements](#indentedstatements)
+  * [Block](#block)
+  * [WhitespaceComment](#whitespacecomment)
+  * [Whitespace](#whitespace)
+
+## Background
+
+> This section is non-normative.
+
+The indented `.sass` syntax uses whitespace to end statements (linebreaks) and
+indicate nesting (indentation). While this syntax is preferred by some authors,
+it presents authoring challenges, specifically around long lists.
+
+## Summary
+
+> This section is non-normative.
+
+In the Sass indented syntax, line breaks always end a statement, which limits
+how authors can format their code. However, the parser can tell from context
+whether a statement can end at a given point. This document proposes that line
+breaks only end statements when a statement can end, and in any other case, a
+line break is treated as continuing white space.
+
+In addition, this proposal adds semicolons to the indented syntax as explicit
+statement ends, and allows curly braces to wrap blocks.
+
+This proposal will make it possible to include line breaks by using
+an order of operations operator, essentially wrapping the expression in parentheses.
```

Comment:
Technically in Sass an "operator" only refers to infix or prefix operators like `+` or `not`. Parentheses are just called "parentheses".